tutoraspects/templates/aspects/apps/superset/pythonpath/openedx_sso_security_manager.py

```diff
@@ -191,7 +190,7 @@ def extra_get_user_roles(self, username, decoded_access_token):
         {{patch("superset-sso-assignment-rules") | indent(8)}}
         return None
 
-    @memoized(watch=("access_token",))
+    @lru_cache(maxsize=LRU_CACHE_MAX_SIZE)
```

Comment:
@bmtcril do you have any concerns about this?

Reply:
It seems ok to me